oojs.js

```diff
@@ -1,12 +1,29 @@
 // Define a Playlist class. It should have...
+class Playlist {
+  constructor(title, songs){
+    this.title = title
+    this.songs = []
+    this.favorites = 0
+  }
+  addSong(songs){
+    this.songs.push(songs)
+  }
+}
   // A title property that is determined by some input.
   // A songs property that is determined by some input. It should contain multiple songs, each of which can just be a song title.
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
 
 // Create an instance of the Playlist class.
-
+var grinding = new Playlist("Emergency", "Swamp Thing")
```

Comment:
The data type for the `songs` argument should be something that can contain multiple songs (i.e., an array).